src/meshapi/views/forms.py

```diff
@@ -37,6 +39,42 @@ class JoinFormRequest:
     ncl: bool
 
 
+class JoinFormRequestSerializer(DataclassSerializer):
+    class Meta:
+        dataclass = JoinFormRequest
+
+
+form_err_response_schema = inline_serializer("ErrorResponse", fields={"detail": serializers.CharField()})
+
+
+@extend_schema_view(
+    post=extend_schema(
+        tags=["User Forms"],
+        auth=[],
+        summary="Register a new request for a potential mesh Install. "
+        "Used by the join form posted on the nycmesh.net website",
+        request=JoinFormRequestSerializer,
+        responses={
+            "201": OpenApiResponse(
+                inline_serializer(
+                    "JoinFormSuccessResponse",
+                    fields={
+                        "detail": serializers.CharField(),
+                        "building_id": serializers.IntegerField(),
+                        "member_id": serializers.IntegerField(),
+                        "install_number": serializers.IntegerField(),
+                        "member_exists": serializers.BooleanField(),
+                    },
+                ),
+                description="This install's building did not already have an install. One has been created and assigned.",
```

Comment:
We should clearly define the behavior of the join form here, and I don't think this is 100% accurate. It will create an Install basically as long as the Member has not created an install at the same building in the same unit before.

Reply:
```suggestion
                description="This install's building did not already have an install for this specific Member in this unit. The Member is deduplicated by Email address. The One has been created and assigned.",
```

Reply:
I don't think either of our summaries is accurate to what we are actually doing in the logic below. It looks to me like we always create the Install object, even if this member already has one on this building. If we want to change this, that work should be captured in a separate issue. For now I'm going to simplify this text to more accurately reflect the current logic:

`"Request received, an install has been created (along with member and building objects if necessary)."`

Reply:
Oh, I guess you're right.

https://github.com/WillNilges/meshdb/blob/main/src/meshapi/views/forms.py#L171

Yeah I don't think that lines up with what we wanted 😂